src/pygama/skm/build_skm.py

```diff
@@ -0,0 +1,256 @@
+"""
+This module implements routines to build the `skm` tier, consisting of skimmed
+data from lower tiers.
+"""
+
+from __future__ import annotations
+
+import json
+import logging
+import os
+
+import awkward as ak
+import numpy as np
+from lgdo import Array, Table, lh5
+from lgdo.lh5 import LH5Store
+
+log = logging.getLogger(__name__)
+
+
+def build_skm(
+    f_evt: str,
+    f_hit: str,
+    f_dsp: str,
+    f_tcm: str,
+    f_skm: str,
+    skm_conf: dict | str,
+    wo_mode="w",
+    skim_format: str = "parquet",
```

Comment:
The default should be LH5 here. I also think that this should be the only supported option, users can use `view_as()` to get the data in some other format.

Reply:
Resolved in 542c53b